standard/sections/clause_7_normative_text.adoc

```diff
@@ -159,9 +159,128 @@ include::../requirements/core/REQ_data_policy_license.adoc[]
 include::../recommendations/core/REC_data_policy_license.adoc[]
 include::../recommendations/core/PER_data_policy_license.adoc[]
 
-==== Data Policy - Rights
+=== Data Policy - Rights
 
 A WCMP record has a `+properties.rights+` property, providing a statement that concerns all rights not addressed
 by the license such as a copyright statement.
 
 include::../recommendations/core/REC_data_policy_rights.adoc[]
+
+=== Distribution Information
+
+==== Overview
+
+A WCMP record has to provide some information regarding how to access and retrieve the data products. This information is specified by the Distribution information and is defined as one of more links to the corresponding data access service. 
+
+==== Examples
+
+- *_Adding the MQTT information_*
+
+Below is an example of the MQTT information added in the record:
+
+[source,json]
+----
+{
+    "associations": [
+        {
+            "rel": "search",
+            "type": "text/html",
+            "title": "WOUDC - Data - Station List",
+            "href": "https://woudc.org/data/stations"
+        },
+        {
+            "rel"  : "WMO-PubSub",
+            "type" : "application/json",
+            "title": "WIS2 MQTT information",
+            "href" : "mqtt://wis2box:wis2box@mosquitto/",
+            "topic": "wis2/mytopic"
+        }
+     ]
+----
+
+- *_Example with the mandatory distribution information_*
+
+Below is provided an example of a WMS service providing access to images.
+
+[source,json]
+----
+{
+            "rel": "item",
+            "type": "image/png",  <== What about the type when multiple types are available ?
```

Comment:
If multiple formats are available, then this is a function of the URI template and the `variables` (so in the WMS case it is enumerating the various `format` options)

Reply:
yes ok but what type should we put. There is no generic image mime-type. 
So then we put data ?

Reply:
Should we suggest that the provider's primary/default/preferred format SHOULD be provided?